editor/src/clj/editor/properties_view.clj

```diff
@@ -553,10 +557,22 @@
     ignore-alpha (drop-last 2)
     :always (apply str "#")))
 
-(defmethod create-property-control! types/Color [edit-type {:keys [color-dropper-view]} property-fn]
+(defn- save-colors!
+  [^ColorPicker color-picker prefs]
+  (->> (.getCustomColors color-picker)
+       (mapv #(color->web-string % false))
+       (prefs/set! prefs saved-colors-prefs-path)))
+
+(defn- get-saved-colors
+  [prefs]
+  (->> (prefs/get prefs saved-colors-prefs-path)
+       (mapv #(Color/valueOf ^String %))))
+
+(defmethod create-property-control! types/Color [edit-type {:keys [color-dropper-view prefs]} property-fn]
   (let [wrapper (doto (HBox.)
                   (.setPrefWidth Double/MAX_VALUE))
         pick-fn (fn [c] (set-color-value! property-fn (:ignore-alpha? edit-type) c))
+        saved-colors (when prefs ^Collection (get-saved-colors prefs))
```

Comment:
Hmm, I think `prefs` must not be `nil` here, is that right? Saving them does not perform a `nil` check in `save-colors!`, so either both places should check for `nil`, or neither.